src/SlashCommands.tsx

```diff
@@ -1129,6 +1130,23 @@ export const Commands = [
         },
         category: CommandCategories.advanced,
     }),
+    new Command({
```

Comment:
I'm moderately surprised the command isn't behind an `isEnabled` check as well - is there something we can use to disable the command in unintentional environments?

Reply:
Huh, I didn't even notice isEnabled(). We could use the check for native/virtual lookup support on the server. I'm sort of surprised we're ever disabling commands though? Maybe for disabled labs features, but having command just disappear in certain situations rather than just return an error seems really confusing. Especially the 'op' command? I guess this seems to be the precedent though...

Reply:
commands disappearing is definitely a concern, though I think for features like this it's moderately more confusing to have it show up compared to something like `/op` disappearing.